posthog/api/feature_flag.py

```diff
@@ -113,8 +113,135 @@ def allow_request(self, request, view):
         return super().allow_request(request, view)
 
 
+class EvaluationTagSerializerMixin(serializers.Serializer):
+    """
+    Serializer mixin that handles evaluation tags for feature flags.
+    Evaluation tags mark which organizational tags also serve as runtime evaluation constraints.
+
+    Note: SDK clients must send 'evaluation_environments' in their flag evaluation requests
+    for these constraints to take effect. Without this parameter, all flags are evaluated
+    regardless of their evaluation tags.
```

Comment:
the SDK changes associated with this change will come next – they'll let users config the parameters that they'll send to the `/flags` endpoint